.github/ISSUE_TEMPLATE/general.yml

```
@@ -0,0 +1,71 @@
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output
+      render: shell
+
+  - type: textarea
+    id: additional-info
+    attributes:
+      label: Additional Information
+      description: Add any other context about the problem here
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)